src/login-screen.js

```diff
@@ -1,8 +1,44 @@
 import React from 'react';
+import { Link } from 'react-router-dom';
+import { connect } from 'react-redux';
 
-let LoginScreen = () =>
-    <div>
-        <h1>Login here!</h1>
-    </div>
+class LoginScreen extends React.Component {
+    constructor(props){
+        super(props);
+        this.state = {
+            username: '',
+            password: '',
+        }
+    }
 
+    render() {
+        return <div>
+            <h1>Please Login</h1>
+            <form 
+            onSubmit={event => {
+                event.preventDefault();
+                this.props.dispach({
```

Comment:
On line 20, it looks like there's a typo on `dispach`. Could we possibly change this to `dispatch`?